src/Process/Conversion/ConversionProcessBuilder.php

```diff
@@ -73,9 +74,11 @@ protected function newCommonMarkConverter(RootConfig $config)
             $environment->addExtension(new $extension());
         }
 
-        return new Converter(
-            new DocParser($environment),
-            new HtmlRenderer($environment)
-        );
+        // Finally add common mark extension
+        $environment->addExtension(new CommonMarkCoreExtension());
+        $environment->addExtension(new AttributesExtension());
+        $environment->addExtension(new TableExtension());
+
```

Comment:
We can probably remove all these extensions. In that case we can switch between commonmark and GitHub flavoured markdown.